orion.py

```diff
@@ -28,8 +29,10 @@ def cli():
 @click.option("--debug", is_flag=True, help="log level ")
 @click.option("--hunter-analyze",is_flag=True, help="run hunter analyze")
 def orion(config, debug, output,hunter_analyze):
-    """Orion is the cli tool to detect regressions over the runs
+    """
+    Orion is the cli tool to detect regressions over the runs
 
+    \b
```

Comment:
is this needed?

Reply:
Yeah... :( 

https://click.palletsprojects.com/en/8.1.x/documentation/#preventing-rewrapping

We might be able to add empty lines in the code, but i figured the `\b` was simple enough.

Reply:
gotcha, sorry not familiar.